src/Components/Table.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import './Table.css';
+
+
+class Table extends React.Component {
+
+   render() {
+
+      return (
+         <React.Fragment>
```

Comment:
You mentioned in your initial comment that this is all hardcoded for the sake of time, which is totally understandable. Really, I think this is good, as sometimes it's okay to go that route if you're presenting static information. Definitely, if you have a chance to make something dynamic, go for it, but this is functional and serves its purpose.